content/docs/get-started/index.md

```diff
@@ -21,184 +23,142 @@ $ mkdir mlem-get-started
 
-As a basic example, create and execute the following `train.py` Python script:
+Consider the following `train.py`:
 
 ```py
 from sklearn.datasets import load_iris
 from sklearn.ensemble import RandomForestClassifier
+import joblib
 
-from mlem.api import save
-
-
+# load data
 data, y = load_iris(return_X_y=True, as_frame=True)
+
+# instantiate and train model
 rf = RandomForestClassifier(
     n_jobs=2,
     random_state=42,
 )
 rf.fit(data, y)
-save(
-    rf,
-    "models/rf",
-    sample_data=data,
-)
+
+# save to file
+joblib.dump(rf, "models/rf")
 ```
 
-Here, we loaded a well-known
+Here We loaded a well-known
 [Iris flower dataset](https://archive.ics.uci.edu/ml/datasets/iris) with
-`scikit-learn` and trained a simple classifier. Instead of pickling the model,
-we persisted it using MLEM's `save` API.
+`scikit-learn` and trained a simple classifier. Note that `scikit-learn` is just
+an example of many MLEM supported ML frameworks (
+[full list here](/doc/object-reference/model)). Finally, we used `joblib` to
+serialize it to file, so executing the script will dump the model to
+`./models/rf`.
 
-`scikit-learn` is just an example of many supported ML frameworks. Check out the
-[full list here](/doc/object-reference/model).
+</details>
 
-Now, let's see what MLEM saved by using the `tree` terminal command:
+MLEM can analyze the code (or model files), and extract extra metadata which
+will become very useful when using the model.
 
-```cli
-$ tree models/
-models
-├── rf
-└── rf.mlem
+We can either save the model with MLEM to begin with, or use MLEM to import
+existing model file and save the metadata as a side effect.
+
+<toggle>
+<tab title="Save new models">
+
+Just replace the line saving the model with a call to `mlem.api.save`:
+
+```py
+from mlem.api import save
+...
+
+# instead of joblib.dump(rf, "models/rf")
+save(rf, "models/rf", sample_data=df)
 ```
 
-Alongside the model binary `models/rf`, MLEM saved an additional metadata file
-`models/rf.mlem`. We refer to this as a "Codification" of the model. This
-`.mlem` "metafile" contains all the information we need in order to use the
-model later:
+The `sample_data` argument will prove useful later when serving the model. MLEM
+uses it to infer the data schema of the payload automatically!
 
-1. Model methods: Like `predict` and `predict_proba`
-2. Input data schema: Describes the dataframe (Iris dataset)
-3. Python Requirements: `sklearn` and `pandas` in this case, with the specific
-   versions used to train the model
+</tab>
 
-<admon type='tip'>
+<tab title="Import existing models">
 
-Note that we didn't have to specify any of this information ourselves. MLEM
-inspects the object (even if it's complex) and infers all of this automatically!
+You can use `mlem.api.import_object()` to "import" the object from Python code,
+which would create the the `rf.mlem` metadata file:
 
-</admon>
+```py
+import os
 
-<details>
+from mlem.api import import_object
```

Comment:
is this example needed in GS? You can't supply `sample_data` here yet, so looks like `mlem.api.save` is superior. And since it's GS we don't need to tell things exhaustively. Maybe better say "it can be done in Python as well" and give a link to API ref?

Reply:
It made this part make more sense to me. without this it's very awkward because we're mixing saving vs importing with python vs cli. This breaks this "false correlation" so it made the whole thing much more natural to read imo